app/src/main/java/com/android/partagix/model/UserViewModel.kt

```diff
@@ -44,16 +44,13 @@ class UserViewModel(
     }
   }
 
-  private fun setUserToCurrent() {
+  fun setUserToCurrent() {
     val userID = FirebaseAuth.getInstance().currentUser?.uid
 
     if (userID == null) {
       database.getUser("XogPd4oF1nYc6Rag6zhh") { updateUIState(it) }
```

Comment:
Does this make sense to display in any situation? The app shouldn't be usable if not user is linked.
🤷‍♂️

Reply:
Well, it's a safety check, better than having an app that crashes

Reply:
Good point. I mean we should go back to the login screen in this case or something like this but it's ok for now.